src/atoms/Table/Table.stories.js

```diff
@@ -281,11 +281,51 @@ export default (asStory) => {
               <Table.HeaderCell>Updated At</Table.HeaderCell>
               <Table.HeaderCell />
             </Table.Header>
+            <Table.Body loading actionTitle="Create Client" onActionClick={ () => alert('Create') } />
+          </Table.Plate>
+        </div>
+      ))
 
-            <Table.Body>
-              <Loader stretch />
-              <Table.Action onClick={ () => alert('Create') }>Create Client</Table.Action>
-            </Table.Body>
+      .add('without data', () => (
+        <div style={{ display: 'flex', height: '600px' }}>
+          <Table.Plate stretch>
+            <Table.Header columns="repeat(6, 1fr)">
+              <Table.HeaderCell>Id</Table.HeaderCell>
+              <Table.HeaderCell>Name</Table.HeaderCell>
+              <Table.HeaderCell>Email</Table.HeaderCell>
+              <Table.HeaderCell>Created At</Table.HeaderCell>
+              <Table.HeaderCell>Updated At</Table.HeaderCell>
+              <Table.HeaderCell />
+            </Table.Header>
+
+            <Table.Body />
+          </Table.Plate>
+        </div>
+      ))
+
+      .add('with custom action button', () => (
+        <div style={{ display: 'flex', height: '600px' }}>
+          <Table.Plate stretch>
+            <Table.Header columns="repeat(6, 1fr)">
+              <Table.HeaderCell>Id</Table.HeaderCell>
+              <Table.HeaderCell>Name</Table.HeaderCell>
+              <Table.HeaderCell>Email</Table.HeaderCell>
+              <Table.HeaderCell>Created At</Table.HeaderCell>
+              <Table.HeaderCell>Updated At</Table.HeaderCell>
+              <Table.HeaderCell />
+            </Table.Header>
+
+            <Table.Body
+              actionTitle={ (
```

Comment:
@ej9x may be rename `actionTitle` to `action` or something else? `actionTitle` with passed button looks strange

Reply:
fixed